src/de_mut/node_seq.rs

```diff
@@ -9,8 +9,8 @@ use serde::{de, Deserialize};
 /// 迭代 `NodeSeq` 可获得一系列 [`NodeSeqItem`]，再调用 `deserialize` 方法分别解析每个节点。
 pub struct NodeSeq<'de> {
```

Comment:
We'd better fix memory layout using `#[repr(C)]` for `NodeSeq` and `GroupDeserializer`.